src/Microsoft.Azure.SignalR.ServerlessAgent/AccessToken.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Microsoft.Azure.SignalR.ServerlessAgent
+{
+    public class AccessToken
```

Comment:
For SDKs, always use `internal` unless you'd like the class to be used by SDK consumers